doc/sphinx/source/getting_started/config.inc

```diff
@@ -9,14 +9,10 @@ There are several configuration files in ESMValTool:
   - config-user.yml
   - config-developer.yml
   - config-references.yml
-  - environment.yml
+  - config-logging.yml
 
-The ``config-user.yml`` file is described here, however, the
-other three files are beyond the scope of the user guide. They are described
-in the developer guide.
-
-User config file
-================
+User configuration file
```

Comment:
There is a reason why these files are not described here: normal users should not need to know about them.

Reply:
Good point, how about describing `config-user.yml` in "Getting started" and the others in "ESMValTool core" sections?